pagopa/io-app
# chore: [IA-736] Remove instabug

## THIS PR DEPENDS ON #3770 
## Short description
This PR removes instabug from the code base

## List of changes proposed in this pull request
- Removed instabug components, logics and package from the code base

## How to test
Try to use the app both with IOS and Android.

Some points to pay attention:
- if the assistanceTool flag in the devServer is setted on instabug the help button should be not visible
- if the assistanceTool flag in the devServer is set to zendesk nothing should change

Tests affected by the change have been updated
